docs/en/04_Changelogs/4.0.0.md

```diff
@@ -61,6 +61,10 @@ or developing your own website. These improvements are mainly geared at CMS core
    * [react-addons-test-utils](https://www.npmjs.com/package/react-addons-test-utils)
  * Added SilverStripeComponent base class for building React UIs
 
+### Compiling CSS
+
+Compass has been removed and css/sprite compilation is now handled with gulp (see [Markup and Style Conventions](../developer_guides/customising_the_admin_interface/cms_architecture/#markup-and-style-conventions)).
```

Comment:
Link to both Compass and Gulp projects - not every PHP dev will be on top of the latest buzzwords ;)